### DISCUSSION TITLE
What is the best way to re-integrate migration code from third party repositories?

### DISCUSSION THREAD

**Discussion by jcdrubay** [OCA/OpenUpgrade#3262] [Show and tell]
The visibility of the contributors is based on the following:

1/ [CONTRIBUTORS page](https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_scripts/readme/CONTRIBUTORS.rst)

I am not sure if this page is actually published somewhere.

2/ Based on the authors of the commits. But there could be contributors who are not the author of commits.

3/ As a copyright at the top of the files for which someone contributed like [here](https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py]

`# Copyright 2021 ForgeFlow S.L. <https://www.forgeflow.com>`

This discussion has been initiated in this issue: https://github.com/Viindoo/OpenUpgrade/issues/262

**Comment by legalsylvain** [comment]
Hi @jcdrubay. If you want to use code from a fork, please 
- ask to commiters to sign the OCA CLA (if not already donne)
- make manualy PR against according branch. (You can cherry pick commits per module and make a PR per module or make a new commit with author gît attributes)

**Comment by legalsylvain** [comment]
Note : if you dont want to make the second step (or not for all modules, ensure that CLA are OK to make sur other people can finish the work legally.)
